packages/auth/src/api/index.ts

```diff
@@ -181,7 +181,10 @@ export async function _performFetchWithErrorHandling<V>(
     if (e instanceof FirebaseError) {
       throw e;
     }
-    _fail(auth, AuthErrorCode.INTERNAL_ERROR, { 'message': String(e) });
+    // Changing this to a different error code will log user out when there is a network error
+    // because we treat any error other than NETWORK_REQUEST_FAILED as token is invalid.
+    // https://github.com/firebase/firebase-js-sdk/blob/master/packages/auth/src/core/auth/auth_impl.ts#L309-L316
```

Comment:
nit: change to the permalink - https://github.com/firebase/firebase-js-sdk/blob/4fbc73610d70be4e0852e7de63a39cb7897e8546/packages/auth/src/core/auth/auth_impl.ts#L309-L316, so the reference will stay the same even when we made changes to auth_impl.ts

Reply:
Done.